.gitmodules

```diff
@@ -1,15 +1,16 @@
 [submodule "deps/jsstyle"]
-	path = deps/jsstyle
-	url = https://github.com/joyent/jsstyle.git
+    path = deps/jsstyle
+    url = https://github.com/joyent/jsstyle.git
```

Comment:
Not sure about this. I did `git submodule set-branch` and this is what happened. I'll double-check that though and make sure it's not emacs exerting its will rather than git.